Kernel_23/doc/Kernel_23/CGAL/Kernel/global_functions.h

```diff
@@ -803,6 +815,15 @@ template <typename Kernel>
 Comparison_result compare_slopes(const CGAL::Segment_2<Kernel> &s1,
 const CGAL::Segment_2<Kernel> &s2);
 
+/*!
+compares the slopes of the segments `(p,q)` and `(r,s)`,
+with `p.z() >= q.z()` and  `r.z() >= s.z()`.
+*/
+template <typename Kernel>
+Comparison_result compare_slopes(const CGAL::Point_3<Kernel> &p,
+                                 const CGAL::Point_3<Kernel> &q,
+                                 const CGAL::Point_3<Kernel> &r,
+                                 const CGAL::Point_3<Kernel> &s);
```

Comment:
1. I am surprised that the function is named `compare_slopes` (plural) and the kernel functor `Compare_slope_3` (singular).
2. The sentence "<quote>compares the slopes of the segments `(p,q)` and `(r,s)`</quote>" is not a correct documentation of the function. In 2D, the slope is well defined, but not in 3D: it could have been the delta of `x` divided by the length. Maybe the functor should be named `Compare_slope_z_3`... actually I cannot find a good name easily. I suggest to ask on CGAL-develop: that will help the small feature be accepted faster.